sdk/purview/purview-catalog-rest/src/purviewCatalog.ts

```diff
@@ -3,14 +3,16 @@
 
 import { getClient, ClientOptions } from "@azure-rest/core-client";
 import { TokenCredential } from "@azure/core-auth";
+import { ServiceVersion } from "./parameters";
 import { PurviewCatalogClient } from "./clientDefinitions";
 
 export default function createClient(
   Endpoint: string,
+  serviceVersion: ServiceVersion,
   credentials: TokenCredential,
   options: ClientOptions = {}
 ): PurviewCatalogClient {
-  const baseUrl = options.baseUrl ?? `${Endpoint}/catalog/api`;
+  const baseUrl = options.baseUrl ?? `${Endpoint}/catalog/api/atlas/${serviceVersion}`;
```

Comment:
This seems like the right behavior. Did you fix it in the codegen or is this a manual edit?

Reply:
It would be great if we could define in the swagger a default value for serviceVersion. Which would generate the `serviceVersion` parameter inside the options. The generated code could do this

```ts
const baseUrl = options.baseUrl ?? `${Endpoint}/catalog/api/atlas/${options.serviceVersion ?? "v2"}`;
```